src/Microsoft.Health.Fhir.SqlServer/Features/Search/Expressions/Visitors/QueryGenerators/SearchParameterQueryGenerator.cs

```diff
@@ -4,9 +4,9 @@
 // -------------------------------------------------------------------------------------------------
 
 using System;
-using System.Data.SqlClient;
 using System.Diagnostics;
 using System.Text.RegularExpressions;
+using Microsoft.Data.SqlClient;
```

Comment:
Migrating to use Microsoft.Data.SqlClient (see PR [#104](https://github.com/microsoft/healthcare-shared-components/pull/104)).

Reply:
Thanks Robin!